scripts/modifyConfigIfRCVersion.js

```diff
@@ -13,10 +14,25 @@ if (process.env.TRAVIS_TAG) {
     packageJson.publisher = testPublisher;
     packageJson.version = packageJson.version.slice(0, -3);
 
-    // Delete
     delete packageJson.icon;
     delete packageJson.aiKey;
 
     fs.writeFileSync('package.json', JSON.stringify(packageJson, null, 2) + '\n');
+
+
+    // 2. Modify extensionId in files
+    const extensionIdPattern = /vsciot-vscode.vscode-iot-workbench/g;
```

Comment:
Can we use dedicated file for test environment instead of dynamic replacement? It would be more maintainable and less error prone.

Reply:
Establish new task to track this issue. https://dev.azure.com/mseng/VSIoT/_workitems/edit/1644786